test/bdd/bddtests_test.go

```diff
@@ -114,10 +114,11 @@ func runBDDTests(tags, format string) int {
 				}
 			}
 
-			err := os.RemoveAll("./fixtures/wellknown/jws")
-			if err != nil {
-				panic(err)
-			}
+			//e
+			//rr := os.RemoveAll("./fixtures/wellknown/jws")
```

Comment:
Remove this code

Reply:
Only commented it out for testing, uncommented again